.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
+  startingDeadlineSeconds: 100
+  successfulJobsHistoryLimit: 10
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: todoapp
+            image: radial/busyboxplus:curl
+            resources:
+              requests:
+                cpu: "100m"
+                memory: "64Mi"
+              limits:
+                cpu: "200m"
+                memory: "128Mi"
+            command:
+            - /bin/sh
+            - -c
+            - curl todoapp-nodeport.todoapp.svc.cluster.local/api/ready
+          restartPolicy: OnFailure
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_6_working_with_deamonsets_and_jobs/assets/87720583/0dd98592-dd30-487b-86fb-e0769d320424)